laterpay/views/frontend/partials/post/gift/gift-pass.php

```diff
@@ -30,16 +30,36 @@
 if ( empty( $pass_id ) ) {
     return;
 }
+
+$subscription_data = LaterPay_Helper_Subscription::get_subscription_by_id( $pass_id );
+$is_subscription   = ( ! empty( $subscription_data ) && is_array( $subscription_data ) );
+
 ?>
 
-<div class="lp_js_giftCard lp_gift-card lp_gift-card-<?php echo esc_attr( $pass_id ); ?>">
+<div class="lp_js_giftCard lp_gift-card lp_gift-card-<?php echo esc_attr( $pass_id ); ?> <?php echo $is_subscription ? 'lp_gift-card-subscription' : 'lp_gift-card-timepass'; ?> ">
     <h4 class="lp_gift-card__title"><?php echo esc_html( $gift_pass['title'] ); ?></h4>
     <p class="lp_gift-card__description"><?php echo wp_kses_post( $gift_pass['description'] ); ?></p>
     <table class="lp_gift-card___conditions">
         <tr>
             <th class="lp_gift-card___conditions-title"><?php esc_html_e( 'Validity', 'laterpay' ); ?></th>
             <td class="lp_gift-card___conditions-value">
-                <?php echo esc_html( $gift_pass['duration'] . ' ' . $period ); ?>
+                <?php
+                if ( $is_subscription ) {
+                    /* translators: %s: Subscription Period */
+                    $subscription_text = sprintf( _x( 'Subscription (Renews in a %s, cancellable anytime)', 'laterpay' ), strtolower( $period ) );
```

Comment:
Do we need `_x` here? second param for the function accepts the context, which isn't passed here, since the `$period` itself is a translated text, maybe we could just use `__` here.